src/Error.tsx

```diff
@@ -0,0 +1,25 @@
+type Props = {
+  errorMessage: string;
+  setErrorMessage: (newMessage: string) => void;
+};
+
+export const Error: React.FC<Props> = ({
+  errorMessage,
+  setErrorMessage = () => {},
+}) => {
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className={`notification is-danger is-light has-text-weight-normal ${errorMessage ? '' : 'hidden'}`}
```

Comment:
Use [classnames](https://www.npmjs.com/package/classnames) library to add classes conditionally